pkgs/by-name/fa/falkor/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  electron,
+  dpkg,
+  makeWrapper,
+  commandLineArgs ? "",
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "falkor";
+  version = "0.0.92";
+
+  src = fetchurl {
+    url = "https://github.com/Team-Falkor/app/releases/download/v${finalAttrs.version}/falkor.deb";
+    hash = "sha256-yDpYu2ehrRQuD29jcyTQla2R2IT1zfBDeWDDRnmqc8Y=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg -x $src ./
+    runHook postUnpack
+  '';
+
+  nativeBuildInputs = [
+    makeWrapper
+    dpkg
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mv usr/share $out/share
+    sed -i "s|Exec=.*|Exec=$out/bin/falkor|" $out/share/applications/*.desktop
+    mv opt/falkor $out/opt
+    makeWrapper ${lib.getExe electron} $out/bin/falkor \
```

Comment:
Since we're using our own electron, most of the files are not necessary inside the opt directory.
IIRC we only copy `resources/` `resources.pak` and `locales/`.
See other electron apps inside nixpkgs as an example.